### ISSUE TITLE
Is RSS from wildcard URL supported?

### ISSUE THREAD

**Issue by solarchemist** [judell/h_notify#1]
Do you support getting RSS from a wildcard URL (domain), i.e., something like this?
```
notify_rss_url_activity(url='https://theatlantic.com/*', token=default_token, pickle='akVkkdfu')
```

When I tried the above I got `NameError: name 'notify_rss_url_activity' is not defined`. 
I was able to run the RSS recipe in `examples.py` without errors.

On a fresh Ubuntu 18.04 VM, I had to install the following pip packages first:
```
sudo apt install python3-pip
pip3 install feedgen markdown python-dateutil pytz wheel 
```

**Comment by judell** [comment]
No, thanks for pointing this out, the RSS pattern was never fully fleshed out and currently only supports the group facet. I'll aim to generalize that.

**Comment by solarchemist** [comment]
For any future reader who is just looking for a way to get RSS output from the Hypothesis Stream for a wildcard URL, this is actually already supported (as pointed out to me by @judell in an email thread):  
[https://hypothes.is/stream.atom?wildcard_uri=https://elifesciences.org/*](https://hypothes.is/stream.atom?wildcard_uri=https://elifesciences.org/*)

Hypothes.is has a guide on [how to use domain-level search here](https://web.hypothes.is/help/how-to-use-domain-level-search/), and [another on RSS/Atom feeds here](https://web.hypothes.is/help/atom-rss-feeds-for-annotations/). As evidenced by the RSS feed above, it seems the functionalities can also be combined.

**Comment by judell** [comment]
>  it seems the functionalities can also be combined.

Yes. The caveat: no way to authenticate to an
Atom/RSS feed, so the feed works for annotations and replies in the Public
layer but not in private groups. The RSS solution here is mainly for that
purpose.
